.github/workflows/test.yml

```diff
@@ -8,6 +8,7 @@ on:
   pull_request:
     branches:
     - master
+  workflow_dispatch: # for manual triggering
```

Comment:
Dave, this is unrelated to your change; did you include this on purpose?

Reply:
Shoot, that was a leftover from trying to debug the crashes. Let me remove that.